packages/fxa-settings/pm2.config.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
         'src/components/**/*.scss',
         require.resolve('fxa-react/configs/tailwind.js'),
       ],
-      ignore_watch: ['src/styles/tailwind.out.css'],
+      ignore_watch: ['src/styles/tailwind.out.*'],
```

Comment:
Might be worth adding this to the other tailwind projects since they may change to SCSS one day

Reply:
go ahead ;)

Reply:
lol fair. clear to merge this

Reply:
lol fair. clear to merge this